src/lecturer/components/createQuestionView/CreateQuestionView.tsx

```diff
@@ -35,48 +38,135 @@ export function CreateQuestionView(){
                 margin: theme.spacing(1),
             },
         },
+        titleInput:{
+            width: "100%",
+            backgroundColor: "white"
+        },
         textarea: {
             width: "100%",
             backgroundColor: "white"
         },
         right: {
             display: "flex",
             justifyContent: "flex-end"
+        },
+        errorColor: {
+            color: "red"
         }
     })();
 
     const handleInputChange = (e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>, index: number) => {
         const { value } = e.target;
         const list = [...inputList];
-        list[index].answer = value;
-        setInputList(list); 
-        console.log(list);
+        list[index] = value;
+        setInputList(list);
     };
 
+    const handleTitleChange = (e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+        const { value } = e.target;
+        setTitle(value);
+    }
+
+    const handleTextAreaChange = (e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+        const { value } = e.target;
+        setQuestion(value);
+    }
+
+    const handleCheckboxChange = (e: ChangeEvent<any>, index: number) => {
+        const value = e.target.checked;
+        const listcb = [...checked];
+        listcb[index] = value;
+        setChecked(listcb);
+    }
+
     const handleAddButtonClick = () => {
-        setInputList([...inputList, { answer: ""}]);
+        setInputList([...inputList, ""]);
+        setChecked([...checked, false]);
     };
     
     const handleRemoveButtonClick = (index: number) => {
         const list = [...inputList];
-        console.log(list);
         list.splice(index, 1);
-        console.log(list);
         setInputList(list);
     };
 
+    const validate = () => {
+        let temp: any = {};
+        let required: string = "To pole jest wymagane";
+
+        temp.title = title ? "" : required;
+        temp.question = question ? "" : required;
+
```

Comment:
https://tenor.com/view/jojos-bizarre-adventure-hmm-gif-10296214
what are these empty lines for?